File: pset1/mario/more/mario.c
// program that takes in an integer value between 0 and 23 and outputs a full pyramid with a space in the middle

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 0 || height > 23);


    int i, space, hash;

    for (i = 0; i < height; i++)
    {
        for (space = 0; space < height - i - 1; space++)
        {
            printf("%s", " ");
        }

        for (hash = 0; hash <= i; hash++)
        {
            printf("%s", "#");
        }

        printf("%s", "  ");

        for (hash = 0; hash <= i; hash++)
        {
            printf("%s", "#");
        }

        printf("\n");
    }
}

File: pset6/sentimental/mario/less/mario.py
from cs50 import get_int

if __name__ == "__main__":
    while True:
        height = get_int("Height: ")
        if height > 0 and height <= 8:
            break
    
    for i in range(height):
        for space in range(0, height-i-1):
            print(' ', end="")
        print('#', end="")
        for hash in range(0, i):
            print('#', end="")
        print('\n', end="")